bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormConstants.java

```diff
@@ -104,4 +104,12 @@ private FormConstants() {
     public static final String RT_FD_FORM_RESET_BUTTON_V1 = RT_FD_FORM_PREFIX + "actions/reset/v1/reset";
 
     public static final String FORM_FIELD_TYPE = "form";
+    public static final String RT_FD_FORM_RENDER_CONDITION_CUSTOM_PROPERTIES = RT_FD_FORM_PREFIX
+        + "container/v1/renderconditions/customProperties";
+    public static final String CUSTOM_PROPERTY_GROUP_NAME = "customPropertyGroupName";
```

Comment:
In hindsight this may not be the best place to add constants since they are not related to core forms but I need them in a common place since they are reused in multiple places. Please suggest an alternative if this is an issue.

Reply:
If the use is limited to just one class, we should declare them in the class itself.

Reply:
+1